doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -459,7 +459,7 @@ following extension of the binder syntax:
 
 Caveat: ``& T`` and ``of T`` abbreviations have to appear at the end
 of a binder list. For instance, the usual two-constructor polymorphic
-type list, i.e. the one of the standard ``List`` library, can be
+type list, i.e., the one of the standard ``List`` library, can be
```

Comment:
Only 27 of 143 uses of "i.e." in the doc are in the form "i.e.,", but OK to change.